tests/nose2.cfg

```diff
@@ -17,4 +17,11 @@ clear-handlers = True
 log-level = DEBUG
 
 [multiprocess]
-always-on = True
\ No newline at end of file
+always-on = True
+test-run-timeout = 600
```

Comment:
I did not find a lot of hints on this in the docu. Hope it is the same as `process-timeout` in `nosetests`, but am no exactly sure about this.